.eleventy.js

```diff
@@ -30,8 +30,12 @@ module.exports = function (eleventyConfig) {
             return "<pre class=\"highlight\"><code class=\"hljs " + lang + "\">" + markdownit.utils.escapeHtml(str) + "</code></pre>";
         }
     });
+
+    var string = require("string");
```

Comment:
It may make more sense to use the `slugify` package that Eleventy uses for URL generation (see: https://www.11ty.dev/docs/filters/slug/). Note that in Eleventy 1.0, which this site doesn't use yet, they are migrating to `@sindresorhus/slugify`, so maybe adding that dependency and using it will make the Eleventy upgrade cleaner when the time comes).

Reply:
As discussed in the chats, the newer slugify requires ES6, which is way too much work for this effort.  We agreed to use the version bundled with our (old) version of eleventy, and I've confirmed that works fine.